src/components/UserActions/UserActions.ts

```diff
@@ -318,16 +317,33 @@ export class UserActions extends Component {
     private renderNoActions() {
         const element = document.createElement('div');
```

Comment:
For a moment, I was confused to see these two lines together. Didn't notice the `this.` at first.

    element.innerHTML = // the whole no actions markup...
    this.element.innerHTML = '';

I'd suggest to rename the `element` variable to `emptySpace` or something like that to avoid confusion with `this.element`. Feel free to keep the code as is if you prefer though.

Reply:
Yea good idea, I was confused about it too (not my code the element part ;) )